__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -306,9 +305,6 @@ describe('start', () => {
   it('should start a wallet with a transaction history', async () => {
     // Send a transaction to one of the wallet's addresses
     const walletData = precalculationHelpers.test.getPrecalculatedWallet();
-    const injectAddress = walletData.addresses[0];
-    const injectValue = getRandomInt(10, 1);
-    const injectionTx = await GenesisWalletHelper.injectFunds(injectAddress, injectValue);
```

Comment:
With this change, the concept of this test was modified: it was supposed to validate if a wallet _already containing transactions_ in its history would be loaded correctly.

In the way it's built now, it starts _an empty wallet_ just like the test above.

The way I see it, we could either:
- Reformulate the test to stop the wallet and start it again ( we would need to get its precalculated data and store it )
- Remove this test, as it is a behavior already tested by the genesis wallet and the miner wallets helpers

Reply:
I decided to continue with the test using another method to send the transaction

https://github.com/HathorNetwork/hathor-wallet-lib/pull/590/commits/4ec53859fa1be9798162eb0f735d7a83875e0d7b

What do you think?